pkg/providers/alibaba/objectstore/objectstore.go

```diff
@@ -115,6 +116,15 @@ func (o *objectStore) CheckBucket(bucketName string) error {
 
 // DeleteBucket deletes a bucket from the object store
 func (o *objectStore) DeleteBucket(bucketName string) error {
+	obj, err := o.ListObjects(bucketName)
+	if err != nil {
+		return emperror.With(emperror.Wrap(err, "could not list objects"), "bucket", bucketName)
```

Comment:
use emperror's WrapWith insted; the woding should be "faieled to list objects" (to be consistent across providers)